utils_cv/detection/bbox.py

```diff
@@ -1,7 +1,11 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-from typing import List, Union
+from .mask import binarise_mask
```

Comment:
[minor] i think the convention we've been using has to been to put references to our own lib at the end of the import statements.